### ISSUE TITLE
Reload content pop

### ISSUE THREAD

**Issue by iBet7o** [sandywalker/webui-popover#195]
Hi!

Is there any way to load content when the pop is open?

Regards!

**Comment by IhorTk** [comment]
Same question

**Comment by xscence** [comment]
Im wondering the same...

**Comment by sandywalker** [comment]
I've added the new API method 'updateContent' , now I'm doing some tests, please wait .

**Comment by xscence** [comment]
It will be great yes. To update the content while popup is opened! This will be very helpful. Thanks!

**Comment by chanusukarno** [comment]
Yes please roll it out! currently am using jquery to replace the popover contents on: `onShow `Event, which is creating weird positioning problems.

**Comment by chanusukarno** [comment]
@sandywalker : When are you rolling out the 'updateContent' feature??

**Comment by sandywalker** [comment]
Sorry for delayed so long.
I've added the global method ```WebuiPopovers.updateContent(selector,newContent) ``` , now you can update the popover content when the pop is open. 
The demo code is in demo/test-issue195.html.

**Comment by chanusukarno** [comment]
`updateContent `works fine!! Thanks!

But with the latest update to `"webui-popover": "1.2.14"` , popover is closing on hover even when `closable `set to true. In earlier version it was fine.

**Comment by xscence** [comment]
Great. Now will replace the bootstrap popover with this one again. Thanks!

**Comment by sbthemes** [comment]
It seems this feature not working with async type.
Any way to achieve this? I have ajax popover and want to update content with other ajax request or even inline content will also work as I will do ajax call myself and will try to replace content but it seems not working here.

**Comment by sandywalker** [comment]
@sbthemes I'm implementing a new global method 'updateContentAsync' and doing some test now, it will be released soon after tests.

**Comment by sbthemes** [comment]
Ok Thanks :)
